src/coverage.rs

```diff
@@ -2,41 +2,28 @@ use std::env;
 use std::fs::OpenOptions;
 use std::io::Write;
 use std::ops::Range;
-use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::atomic::{AtomicUsize, Ordering};
 
-lazy_static! {
-    static ref COVERAGE_ALREADY_CHECKED: Vec<AtomicBool> = {
-        let max = env::var("MUTAGEN_MUTATION_AMOUNT").map(|s|s.parse().unwrap_or(0)).unwrap_or(0) + 1;
```

Comment:
On the runner, we are still passing this envvar. We should remove this from the runner code.

Reply:
Agreed. :+1: